pkg/vtpm/build.yml

```diff
@@ -10,7 +10,7 @@ config:
   binds:
     - /dev:/dev
     - /run:/run
-    - /config:/config
+    - /persist:/persist
```

Comment:
Would it make sense in a later PR to pick subsets of /run and /persist to minimize what the vtpm container can access?

Reply:
I vaguely remember that it didn't work, let me try again and if worked I will change it.

Reply:
fixed.